draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -648,6 +648,34 @@ Here, `eyJhbGci...emhlaUJhZzBZ` represents the SD-JWT and `eyJhb...dYALCGg` repr
 
 Other specifications or profiles of this specification may define alternative formats for transporting the Combined Format for Presentation that envelopes multiple such objects into one object, and provides Key Binding using means other than the Key Binding JWT.
 
+# JWS JSON Serialization {#json_serialization}
+
+(Work in progress...)
+
```

Comment:
```suggestion
```

Reply:
I added this just to show that even the sentences I wrote were just placeholders. I think we can remove this for now.

Reply:
I left it in for similar reasons but can remove at this point.